lib/ansible/modules/windows/win_chocolatey.py

```diff
@@ -51,6 +51,14 @@
   source:
     description:
       - Specify source rather than using default chocolatey repository.
```

Comment:
I would add here a statement to indicate this is actually a method for using *stateless* sources (so without modifying the running configuration). I think we need to make this absolutely clear, because this is a very different use-case then what we (in configmgmt) would recommend.

Personally I think this way of working is more a convenience, rather than a proper way of working. We would need a `win_chocolatey_source` module for the proper implementation.